src/vorta/store/settings.py

```diff
@@ -59,6 +59,14 @@ def get_misc_settings() -> List[Dict[str, str]]:
             'label': trans_late('settings', 'Get statistics of file/folder when added'),
             'tooltip': trans_late('settings', 'When adding a new source, calculate its size and the number of files.'),
         },
+        {
+            'key': 'enable_fixed_units',
+            'value': False,
+            'type': 'checkbox',
+            'group': information,
+            'label': trans_late('settings', 'Display all archive sizes in a consistent unit of measurement'),
+            'tooltip': trans_late('settings', 'When enabled, replaces dynamic units of measurement based on file size'),
```

Comment:
Do you mean _archive size_ instead of *file size*?

Reply:
Definitely - thanks for catching that! I will correct on next commit.